dev/logging/MonologLoader.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace SilverStripe\Framework\Dev\Logging;
+
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+use Monolog\ErrorHandler;
+
+
+/**
+ * Adaptor connecting Monolog to SilverStripe for error handling
+ */
+class MonologLoader
+{
+	function __invoke() {
+		// Register the default error handler
+		$logger = \Injector::inst()->get('Logger');
+		ErrorHandler::register($logger);
+
+		// Load the error handlers into the monolog loader
```

Comment:
MonologLoader was intended to be a factory. Perhaps making it callable is a problem?